mods/d2k/rules/arrakis.yaml

```diff
@@ -1,22 +1,25 @@
 spicebloom.spawnpoint:
-	HiddenUnderShroud:
-		Type: CenterPosition
+	EditorOnlyTooltip:
+		Name: Spice Bloom spawnpoint
+	AlwaysVisible:
+	RenderSpritesEditorOnly:
+		Image: spicebloom
```

Comment:
For extra polish I suggest using full bloom frame (right now it's only one pixel) and the `effect50alpha` palette like we do for TS destroyed bridges.